src/current/v23.1/sso-sql.md

```diff
@@ -1,158 +1,257 @@
+|`server.oidc_authentication.generate_cluster_sso_token.use_token`| This selects which part of the received OIDC credentials to use to [determine the user’s identity](#how-cockroachdb-determines-the-sql-username-from-a-jwt). Normally this should be set to `id_token`.
+|`server.identity_map.configuration`| Takes an [Identity Map configuration](#identity-map-configuration)
+| `server.sql_host` | This display value informs users the host for their SQL connections. Default: `localhost`.
+| `server.sql_port` | his display value informs users the port for their SQL connections. Default: `26257`.
 
-### `server.jwt_authentication.enabled`
+### Update your cluster settings
 
-	Enable or disable Cluster SSO
+You can update your cluster settings with the [`SET CLUSTER SETTING`]({% link {{ page.version.version }}/set-cluster-setting.md %}) SQL statement.
 
-	Required value: `true`
+You can also view all of your cluster settings 
 
-	Check to confirm that JWT authentication is enabled on your cluster. It is enabled by default in CockroachDB {{ site.data.products.cloud }} clusters.
+1. `enable` JWT SQL authentication to your cluster.
 
-### `server.jwt_authentication.jwks`
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    SET CLUSTER SETTING server.jwt_authentication.enabled = true;
+    ~~~
 
-Add your IdP's public signing key to your cluster's list of accepted signing JSON web keys (JWKS), under the `jwks` setting. This is a [JWK](https://www.rfc-editor.org/rfc/rfc7517) formatted single key or key set, containing the public keys for SSO token issuers/IdPs that will be accepted by your cluster.
+1. Add your IdP's public signing key to your cluster's list of accepted signing JSON web keys (JWKS), under the `jwks` setting. This is a [JWK](https://www.rfc-editor.org/rfc/rfc7517) formatted single key or key set, containing the public keys for SSO token issuers/IdPs that will be accepted by your cluster.
 
-By default, your cluster's configuration will contain the CockroachDB {{ site.data.products.cloud }}'s own public key, allowing CockroachDB {{ site.data.products.cloud }} to serve as an IdP. When modifying this cluster setting, you must include the CockroachDB {{ site.data.products.cloud }} public key in the key set. Failing to do so will lock out console SSO users and prevent maintenance access by the CockroachDB {{ site.data.products.cloud }} managed service, leading to unintended consequences.
+    By default, your cluster's configuration will contain the CockroachDB {{ site.data.products.cloud }}'s own public key, allowing CockroachDB {{ site.data.products.cloud }} to serve as an IdP. When modifying this cluster setting, you must include the CockroachDB {{ site.data.products.cloud }} public key in the key set. Failing to do so will lock out console SSO users and prevent maintenance access by the CockroachDB {{ site.data.products.cloud }} managed service, leading to unintended consequences.
```

Comment:
This section is mostly wrong. In cloud if you use ccloud then the key will be included. If you change it then ccloud sso login won't work.

Reply:
better?

Reply:
This is looking the same to me?